script.js

```diff
@@ -1,4 +1,154 @@
+var user_lose = 0;
+var user_win = 0;
+var user_draw = 0;
+var current_mode = "waiting";
+var user_name = "";
+var myOutputValue = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if (current_mode == "waiting") {
+    user_name = input;
+    current_mode = "gametime";
+    return (myOutputValue =
+      " hi " + user_name + "<br>" + "it's " + current_mode);
+  } else if (current_mode == "gametime") {
+    return (myOutputValue = playgame(user_name, input));
+  }
+};
```

Comment:
Good job keeping your main function short by taking out the game logic into another function!